common_test.go

```diff
@@ -33,7 +31,7 @@ func TestBuildUrl(t *testing.T) {
 					assert.Equal(t, "/1/auth", r.URL.Path)
 					assert.Equal(t, []string{tc.APIKey}, r.Header["X-Honeycomb-Team"])
 
-					if isClassic(tc.APIKey) {
+					if tc.Classic {
```

Comment:
This felt a little too close to using the function under test to make testing decisions, so I updated this check to live in the test case instead.